sentry-types/src/protocol/envelope.rs

```diff
@@ -213,12 +215,14 @@ impl Envelope {
     where
         I: Into<EnvelopeItem>,
     {
+        let item = item.into();
+
         let Items::EnvelopeItems(ref mut items) = self.items else {
-            eprintln!("WARNING: This envelope contains raw items. Adding an item is not supported.");
+            if item != EnvelopeItem::Raw {
```

Comment:
How is `cargo fmt` not complaining here? 👀

Reply:
because `let else` is still not supported after all this time :-(